src/App.js

```diff
@@ -11,15 +21,50 @@ class App extends React.Component {
       cardDeck: makeShuffledDeck(),
       // currCards holds the cards from the current round
       currCards: [],
+      playerOneRoundsWon: 0,
+      playerTwoRoundsWon: 0,
+      roundWinner: "",
+      roundStarted: false,
+      endOfRound: false,
+      roundsPlayed: 0,
     };
   }
 
+  resetRound = () => {
+    this.setState(() => ({
+      cardDeck: makeShuffledDeck(),
+      currCards: [],
+      playerOneRoundsWon: 0,
+      playerTwoRoundsWon: 0,
+      roundsPlayed: 0,
+      endOfRound: false,
```

Comment:
We could probably make an "initialState" object, to not need to repeat this code.